### ISSUE TITLE
No comments in A2C code, very hard to read

### ISSUE THREAD

**Issue by Kagaratsch** [openai/baselines#285]
Would it be possible to make the A2C code a bit more pedagogical?

**Comment by andrewgough94** [comment]
I agree some documentation would be nice, particularly with the output that is logged, can anyone explain what the logged output variables mean? Why raw score isn't there?

**Comment by btaba** [comment]
It looks like the raw env outputs are logged to a directory in `/tmp` created in [`monitor.py`](https://github.com/openai/baselines/blob/master/baselines/bench/monitor.py#L29), which you can tail. It looks like they can be loaded after the fact with [this](https://github.com/openai/baselines/blob/master/baselines/bench/monitor.py#L102).
